apps/src/templates/rubrics/introjs.scss

```diff
@@ -445,4 +445,3 @@ tr.introjs-showElement > th {
   z-index: 1;
   opacity: 0;
 }
-/*# sourceMappingURL=introjs.css.map */
```

Comment:
I've noticed these comments getting automatically removed by the linter in some of my own PRs; I'm not entirely sure why they're being removed because they don't actually appear to violate any linting rules, but I do think we want to preserve them.

Reply:
@Nokondi has suggested we let it be removed. I'm going to keep this PR as is.

Reply:
Are we confident we don't want this comment? Note this isn't just an arbitrary comment; it's intended to enable sourcemapping for this compiled asset. This is the kind of thing where if we remove it, it won't break it an obvious way but may make it difficult down the line to diagnose something else that's broken.

Of course, it could already be broken for all I know :upside_down_face: